keps/168-2-pending-workloads-visibility/README.md

```diff
@@ -257,17 +259,21 @@ We plan on adding sanity e2e tests, and RBAC e2e tests. The e2e RBAC tests shoul
 
 ### Graduation Criteria
 
-First iteration (0.6):
+#### Alpha
 - Release the new API in alpha. This allows us to adjust the API according to users' and reviewers' feedback,
 - Release it with a feature gate.
 
-Second iteration (0.7):
+#### Beta
+First iteration (0.9):
 - Release the API in beta and guarantee backwards compatibility,
-- Reconsider introducing a throttling mechanism based on user and review feedback,
-- Consider introducing FlowScheme and PriorityLevelConfiguration to allow admins to easily tune API priorities.
```

Comment:
@tenzen-y  I think we can still consider this for GA, but I would wait for the user's feedback indicating the `global-default` is not enough. The good thing is that if this happens then still the user can introduce its own custom schema relatively easily without recompiling. AFAIK `metrics-server` also introduces endpoints, but does not provide a dedicated FlowSchema, while being a mature project.

Reply:
For "Reconsider introducing a throttling mechanism based on user and review feedback", I agree with introducing based on use feedback.

For "Consider introducing FlowScheme and PriorityLevelConfiguration to allow admins to easily tune API priorities.", I am still wondering if we can introduce before we graduate to Beta.

This dedicated FlowScheme is not complex task, and we can just YAML file. Additionally, when the visibility endpoints occupy the global-defaults seats, other components and cluster admins are locked out, and then the situation brings us cluster disruption.

So, I would like to introduce this at least guard (dedicated FlowScheme) as much as possible before we enable this feature by default.

Reply:
I see, I'm ok to be a bit more pro-active and provide some default FlowSchema. 

So, I'm ok moving "Introduce FlowScheme and PriorityLevelConfigurationt for visibility endpoints to allow admins to easily tune API priorities." to Beta.

Reply:
I have had some discussions with @mimowo for FlowSchema offline.

In conclusion, we confirmed that GKE does not have special parameters for FlowSchema, and the dedicated FlowSchema allows us to prevent the situation where the visibility server occupies the global default seats.

So, we would like to propose introducing the dedicated FlowSchema with the priority parameters the same as `leader-election` since we can assume that the visibility server would not use the kube-api capacities as much as the global default before we graduate this feature to Beta.

Additionally, it would be great to document how we can relax the kube-api seat limitations for the visibility servers.

If you have any other comments, please let me know. @mimowo

Reply:
I'm ok with the plan @mbobrovskyi please update the KEP criteria for Beta.

Reply:
Actually, I think we probably want to increase the `lendablePercent` for visibility, as this is not a critical endpoint. I think if more important endpoints are indeed we should land. I think conceptually it is like `workload-low` level, so I would give it:
```yaml
spec:
  limited:
    lendablePercent: 90
    limitResponse:
      queuing:
        handSize: 4
        queueLengthLimit: 50
        queues: 16
      type: Queue
    nominalConcurrencyShares: 10
  type: Limited
```
WDYT @tenzen-y ?